packages/angular_devkit/schematics/BUILD

```diff
@@ -28,18 +32,46 @@ ts_library(
     ],
 )
 
+ts_library(
+    name = "schematics_test_lib",
+    srcs = glob(
+        include = [
+            "src/**/*_spec.ts",
+            "src/**/*_spec_large.ts",
+        ],
+    ),
+    deps = [
+        ":schematics",
+        ":testing",
+        "//packages/angular_devkit/core",
+        "//packages/angular_devkit/core:node",
+        "@rxjs",
+        "@rxjs//operators",
+        # @typings: jasmine
+        # @typings: node
+    ],
+)
+
+jasmine_node_test(
+    name = "schematics_test",
+    srcs = [":schematics_test_lib"],
+)
+
+
+# @angular-devkit/schematics/tasks
+
 ts_library(
     name = "tasks",
     srcs = glob(
         include = ["tasks/**/*.ts"],
         exclude = [
             "tasks/**/*_spec.ts",
             "tasks/**/*_benchmark.ts",
+            "tasks/tslint-fix/test/**/*",
         ],
     ),
     module_name = "@angular-devkit/schematics/tasks",
-    module_root = "tasks",
-    tsconfig = "//:tsconfig.json",
+    module_root = "tasks/index.d.ts",
```

Comment:
Changing it to index.d.ts will hide the tasks/node, is that on purpose?

Reply:
I'm not sure what you mean, can you elaborate? I didn't mean to hide any package.

Reply:
tasks/index.ts only export all the tasks, the task executors are exported through tasks/node
Changing module_root to "tasks/index.d.ts" will make tasks/node is not reachable.

Reply:
@qiyigg https://github.com/angular/angular-cli/pull/12008 should make them available again, sorry about that.

Reply:
Cool, thanks!